result/result.py

```diff
@@ -254,12 +254,14 @@ class UnwrapError(Exception):
     Exception thrown upon `.unwrap_<...>` and `.expect_<...>` calls
     """
```

Comment:
Your PR description and explanation for using `Any` was quite good and clear. Would it make sense to include it in the docstring here?

Reply:
i've updated the docstring. please have a look and perhaps suggest additional improvements  🙏